src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -5,14 +5,15 @@ import Highcharts from 'highcharts';
 import kebabCase from 'lodash/kebabCase';
 import merge from 'lodash/merge';
 import numeral from 'numeral';
-import AboutTooltip from '../AboutTooltip/AboutTooltip';
```

Comment:
I ditched this, it does nothing and now there's two `<Tooltip>`s in the chart, it makes no sense to have a special component for one tooltip but not the other.